wnprc_billing/resources/views/updateProgramIncomeAccount.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body onload = "getValues()">
+
+    <table id="myTable" style="border: 1px solid black; font-size: 15px; width: 800px; text-align: center">
+        <tr>
+            <th style="padding:5px; font-size:20px; text-align: center; text-decoration: underline">Variable</th>
+            <th style="padding:5px; font-size:20px; text-align: center; text-decoration: underline">Current Value</th>
+            <th style="padding:5px; font-size:20px; text-align: center; text-decoration: underline">New Value</th>
+            <th></th>
+        </tr>
+        <tr>
+            <td style="padding:5px">CreditToAccount</td>
+            <td style="padding:5px" id="ctaCell1"></td>
+            <td style="padding:5px" id="ctaCell2">
+                <form>
+                    <input type="text" id="field1" name="field1">
+                </form>
+            </td>
+            <td style="padding:5px">
+                <button id="field1Button" onclick="updateField1()">Update CreditToAccount</button>
+            </td>
+        </tr>
+        <tr>
+            <td style="padding:5px">JETCSVSetting</td>
+            <td style="padding:5px" id="jetcsvCell1"></td>
+            <td style="padding:5px" id="jetcsvCell2">
+                <form>
+                    <input type="text" id="field2" name="field2">
```

Comment:
can these fields be given a better id and name too? It just makes it easier when coming back to project to read it.

Reply:
Definitely.  I'll change the following:

field1 -> newCreditToAccountField
field2 -> newJETCSVSettingField
field1Button -> updateCreditToAccountButton
field2Button -> updateJETCSVSettingButton